### DISCUSSION TITLE
阿里云盘OPEN 报错failed to refresh token: The string ' doesn't have a valid JSON format.

### DISCUSSION THREAD

**Discussion by tmzg0000** [AlistGo/alist#4295] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

v3.16.3 和 v3.15.1

### Driver used / 使用的存储驱动

阿里云盘OPEN

### Describe the bug / 问题描述

挂载阿里云盘open 其中一个帐号显示‘ failed to refresh token: The string '{"sub":"68da2d23c0e843f4a0|th�l}�.CosFviָRe�h�fX���� X��M M�L �� ������^��M�L �LL�M��X]��M��L�L�M�' doesn't have a valid JSON format. ’ 中间就是这样的乱码。 其他账号就能成功挂载。想咨询下什么原因导致的？

### Reproduction / 复现链接

抱歉，没有连接

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by tmzg0000** [comment]
就没有人遇到相同的问题吗？

**Reply by liang0222** [reply]
有 我现在就是 未解决

**Reply by txj1996** [reply]
今天阿里云盘OPEN全部挂了！

**Reply by ezjanqun** [reply]
今天阿里云盘Open也打不开了，不知道什么时候能解决？

**Reply by jlifeng** [reply]
更换Oauth令牌链接
https://sni_api_nn_ci.cooluc.com/alist/ali_open/token

**Reply by altairbow** [reply]
> 更换Oauth令牌链接 https://sni_api_nn_ci.cooluc.com/alist/ali_open/token

正解，https://api.nn.ci/alist/ali_open/token 这个地址被墙了

**Reply by PolarBearbilbil** [reply]
> 更换Oauth指令牌链接 https://sni_api_nn_ci.cooluc.com/alist/ali_open/token

正解，已更换令牌链接就可以了
